discovery_engine/test/integration/init_engine_test.dart

```diff
@@ -40,23 +41,17 @@ void main() {
     });
 
     test('init engine with ai models', () async {
-      server = await LocalNewsApiServer.start();
-      final engine = await DiscoveryEngine.init(
-        configuration: createConfig(data, server.port),
-      );
-
+      final engine = await initEngine(data, server.port);
```

Comment:
this line could be moved to the `setUp()` like in the other tests. similarly below.

Reply:
unfortunately, that will break the second test case. it will pass but we don't check if the initialisation is   successful in case of an api error . but we can move the dispose into `tearDown()`